core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/order/service/call/ConfigurableOrderItemRequest.java

```diff
@@ -44,6 +44,8 @@
     protected boolean firstExpandable;
     protected boolean lastExpandable;
 
+    protected boolean priceIncludedInParent;
```

Comment:
Shouldn't we just expose the PricingModelType?  That would be a more flexible approach should a customer need to add special logic on the frontend.

Reply:
I could not find a way to get access to PricingModelType on Site. Because it is in ProductAddOnXref.

Reply:
Added `pricingModelType `to `ConfigurableOrderItemRequest` as a String